docs/nodes/full-storage-node.mdx

```diff
@@ -88,50 +92,84 @@ A note on ports:
   after the IP address or use the  `--core.grpc.port` flag to specify another
   port if you prefer.
 
-````mdx-code-block
-<Tabs groupId="network">
-<TabItem value="mocha" label="Mocha">
-
 ```sh 
-celestia full start --core.ip <ip-address>
+celestia full start --core.ip <ip-address>:<port>
 ```
 
-</TabItem>
-<TabItem value="arabica" label="Arabica 🏗️">
```

Comment:
Why are tabs being removed from here?

Reply:
same as above. this is an old PR and I had to rearrange the order of the tabs, the same way that they are on the live site. where mocha is first, and arabica second